### PR TITLE
Export excel.

### PR DESCRIPTION
**Pull request by JeldorPKU**
Fix the export feature.

### CODE REVIEW

File: application/controllers/admin.php [hunk 1]
```diff
@@ -255,10 +256,11 @@ public function export() {
                 ->setCellValue('A' . $i, $i - 1)
                 ->setCellValue('B' . $i, $item['name'])
                 ->setCellValue('C' . $i, $GLOBALS['GENDER'][$item['gender']])
-                ->setCellValue('D' . $i, "'" . $item['id_number'])
-                ->setCellValue('E' . $i, $school['school'])
-                ->setCellValue('F' . $i, $item['tel'])
-                ->setCellValue('G' . $i, $GLOBALS['ACCOMMODATION'][$item['accommodation']]);
+                ->setCellValue('D' . $i, $GLOBALS['ID_TYPE'][$item['id_type']])
+                ->setCellValue('E' . $i, "'" . $item['id_number'])
```
Comment (huxuan) [inline application/controllers/admin.php]: What is the `"'" .` for?

File: application/views/user_result.php [hunk 2]
```diff
@@ -36,51 +36,47 @@
...
-            <th>住宿</th>
-            <th>5.14晚餐</th>
-            <th>5.15午餐</th>
-            <th>清真</th>
-            <th>费用</th>
+          <th>序号</th>
+          <th>姓名</th>
+          <th>性别</th>
+          <th>手机号</th>
+          <th>证件类型</th>
```
Comment (huxuan) [inline application/views/user_result.php]: Thanks for remembering this. [This is not an issue.]

File: application/views/user_result.php [hunk 3]
```diff
@@ -36,51 +36,47 @@
...
-            <th>证件编号</th>
-            <th>北大赛</th>
-<!--            <th>公路日</th>-->
-<!--            <th>山地日</th>-->
-            <th>住宿</th>
-            <th>5.14晚餐</th>
-            <th>5.15午餐</th>
-            <th>清真</th>
-            <th>费用</th>
+          <th>序号</th>
```
Comment (huxuan) [inline application/views/user_result.php]: Please revert back the indentation change to avoid so many unnecessary changes.
Also for the following part.

File: application/controllers/admin.php (hunk 1 above)
Comment (JeldorPKU) [inline application/controllers/admin.php]: That is for Excel to mark this cell as text instead of number, so that the ID number won't be like 1.10e18.

Comment (huxuan) [inline application/controllers/admin.php]: Gotcha.

**Comment by JeldorPKU** [comment]
Go to #245 to see the latest changes.
